docs/examples/dl3_tool_config.json

```diff
@@ -12,9 +12,9 @@
   },
   "DL3Cuts": {
     "global_gh_cut": 0.7,
-    "gh_efficiency": 0.9,
+    "gh_efficiency": 90,
```

Comment:
That's not what I meant. I think having it in `[0, 1]` in the config file makes sense!

Just passe `percentile=100 * (1 - self.gh_efficiency)` when calling pyirf

Reply:
Ok. I will do the same for `theta_containment`